File: Install_Android/app/src/main/java/com/goldenplanet/www/gp_android/MoveActivity.java
package com.goldenplanet.www.gp_android;

import android.app.Activity;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MoveActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_move);

        Intent intent = getIntent();
        String cc = intent.getStringExtra("cc");
        String cn = intent.getStringExtra("cn");
        String cm = intent.getStringExtra("cm");
        String cs = intent.getStringExtra("cs");
        String ck = intent.getStringExtra("ck");
        String ci = intent.getStringExtra("ci");
        String anid = intent.getStringExtra("anid");
        String gclid = intent.getStringExtra("gclid");
        String dclid = intent.getStringExtra("dclid");
        String aclid = intent.getStringExtra("aclid");
        String gmob_t = intent.getStringExtra("gmob_t");


        
    }
}

File: Install_AOSkt/app/src/main/java/com/goldenplanet/www/gp_android/MoveActivity.kt
package com.goldenplanet.Install_AOSkt

import android.app.Activity
import android.os.Bundle

class MoveActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_move)
        val intent = intent
        val cc = intent.getStringExtra("cc")
        val cn = intent.getStringExtra("cn")
        val cm = intent.getStringExtra("cm")
        val cs = intent.getStringExtra("cs")
        val ck = intent.getStringExtra("ck")
        val ci = intent.getStringExtra("ci")
        val anid = intent.getStringExtra("anid")
        val gclid = intent.getStringExtra("gclid")
        val dclid = intent.getStringExtra("dclid")
        val aclid = intent.getStringExtra("aclid")
        val gmob_t = intent.getStringExtra("gmob_t")
    }
}